src/styles/accountPicker.styl

```diff
@@ -47,12 +47,12 @@
                 line-height 1rem
                 color coolGrey
 
+        .col-account-picker-button-status,
         .col-account-picker-button-error
```

Comment:
This should be done via 

`.col-account-picker-button`

and modifiers `.col-account-picker-button--error` and `.col-account-picker-button--status`